packages/cli/src/util/proposerConfig.ts

```diff
@@ -89,6 +94,7 @@ function parseProposerConfigSection(
     builder: {
       gasLimit: overrideConfig?.builder?.gasLimit ?? (gas_limit !== undefined ? Number(gas_limit) : undefined),
       selection: overrideConfig?.builder?.selection ?? builderSelection,
```

Comment:
Noticed we might wanna also improve validation on builder selection
https://github.com/ChainSafe/lodestar/blob/01d47b9a0901c9f03a01ef68c819f4e592cbfd8b/packages/cli/src/cmds/validator/handler.ts#L257

Might be better to define those functions here and import them in the handler instead of the other way around